index.html

```diff
@@ -149,13 +175,17 @@ <h2 class="project-title">Tonic</h2>
               </div>
             </div>
           </article>
+
           <article class="card">
-            <figure>
+            <figure class="mobile_project">
               <img src="images/big_icon/art.png" alt=" " />
             </figure>
+            <figure class="desktop_project">
+              <img src="images/desktop/Portoflio Card/Uber-Navigation.svg" alt=" " />
+            </figure>
             <div class="left-block">
               <div class="project-text">
-                <h2 class="project-title">Multi-Post Stories</h2>
+                <h2 class="project-title">Uber Navigation</h2>
```

Comment:
- [ ] I kindly suggest that you make sure that you make sure the project's picture and the other half containing the project's description switch places with each project (picture is on the left and description on the right for the first project, description on the left and picture on the right for the second project and so on and so forth).

<img width="828" alt="image" src="https://user-images.githubusercontent.com/52934128/201454071-71c2c155-1281-40c2-af8d-5787f67ead80.png">